src/sagas/lists.js

```diff
@@ -47,45 +52,33 @@ function* fetchList(action) {
   }
 }
 
-function* fetchListLessons(action) {
-  const { withViews = false, namespace, ...args } = action.payload;
+function* fetchSectionList(action) {
+  const { namespace, ...args } = action.payload;
 
   const filters      = yield select(state => filterSelectors.getFilters(state.filters, namespace));
   const filterParams = filtersTransformer.toApiParams(filters) || {};
 
   const language = yield select(state => settings.getLanguage(state.settings));
 
   try {
-    const { data: { items, content_units, total } } = yield call(Api.lessons, { ...args, ...filterParams, language });
+    const { data } = yield call(endpointByNamespace[namespace], { ...args, ...filterParams, language });
 
-    let cuIDs;
-    let cuIDsView;
-    let cIDs;
-    const data = { total, items };
+    const { items } = data;
 
-    if (content_units) {
```

Comment:
What is the case of 
```if (content_units) {```
That you don't need any more?

Reply:
backend response was change so no _ content_units_ more